client-reactjs/src/App.js

```diff
@@ -118,6 +119,8 @@ class App extends React.Component {
 
     const getAssets = () => {
       const applicationId = this.props.application?.applicationId;
+      console.log('getting assets');
+      console.log(applicationId);
```

Comment:
Logging app ID on the client side might not be a safe thing. We can get rid of all console.logs in the client